### DISCUSSION TITLE
Sunbird Lern microsite review and feedback.

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Lern/Community#3] [Documentation]
All,
The **[Lern microsite](https://lern.sunbird.org)** has been put through a few iterations over the last few weeks, and we’re calling for team members, partners, adopters and all those who’re interested to come take a look at it. There will be further iterations over coming weeks to incorporate feedback and suggestions that you provide, and the team will ensure continuous progress to make the site increasingly beneficial. 

Please use **[this link](https://github.com/Sunbird-Lern/Community/discussions/categories/documentation)** to send us your thoughts and feedback - all inputs will be processed with due diligence by the team.

A few items that are already being worked on include:
> Inclusion of a few additional reports that are powered by Lern
> Include telemetry details - of events that are raised by the Lern building block
> Completion of functional and installation configurations of all components
> Inclusion of additional API documentation from sunbird.org into the Lern microsite.

[@Krishnaj20](https://github.com/Krishnaj20) [@pramodkvarma](https://github.com/pramodkvarma) [@jagadishbabu](https://github.com/jagadishbabu) [@madhucr](https://github.com/madhucr) [@alok-os](https://github.com/alok-os) [@surendrasinghs](https://github.com/surendrasinghs) [@kameshbhr](https://github.com/kameshbhr) [@SanthoshVasabhaktula](https://github.com/SanthoshVasabhaktula) [@vrayulu](https://github.com/vrayulu) [@mmanimegalai](https://github.com/mmanimegalai) [@maheshkumargangula](https://github.com/maheshkumargangula) [@reshmi-nair](https://github.com/reshmi-nair) [@sharathkashyap](https://github.com/sharathkashyap) [@vinukumar-vs](https://github.com/vinukumar-vs) [@gandham-santhosh](https://github.com/gandham-santhosh) [@anandp504](https://github.com/anandp504) [@chitranshu-keshav](https://github.com/chitranshu-keshav) [@Prerna789](https://github.com/Prerna789) [@coolbung](https://github.com/coolbung) [@vijiurs](https://github.com/vijiurs) [@aks30](https://github.com/aks30) [@kiranharidas187](https://github.com/kiranharidas187) [@aayushkedia95](https://github.com/aayushkedia95) [@ChakshuGautam](https://github.com/ChakshuGautam) [@vaivk369](https://github.com/vaivk369) [@ahghatol](https://github.com/ahghatol) [@Nadiya496](https://github.com/Nadiya496) [@ashokreddy1208](https://github.com/ashokreddy1208) [@rhwarrier](https://github.com/rhwarrier)

Thanks,
Team Lern

**Comment by pramodkvarma** [comment]
Very good start indeed! Went through it, while there is a lot of content, I suppose this needs overall upgrade in navigation, better explanation, and ease of adoption. This is one of the core BBs behind Sunbird-ED and folks who want to build an alternate UX (say, someone needs to replace their MOOC backend with Sunbird BBs), need to have better documentation to be able to do that. This is key. @rhwarrier @jagadishbabu

Nevertheless, great to see these coming together! Thank you to the team!
